src/core/core.cpp

```diff
@@ -1070,7 +1074,7 @@ void Core::sendGroupMessageWithType(int groupId, const QString& message, Tox_Mes
     QMutexLocker ml{&coreLoopLock};
 
     int size = message.toUtf8().size();
```

Comment:
Can size() return negative values? It may be better to choose the other direction and use size_t for both.